operator/Dockerfile

```diff
@@ -1,8 +1,8 @@
 # We have to emulate directory layout as in the repo so that imports in go files work fine.
 ARG roxpath=/workspace/src/github.com/stackrox/rox
-ARG GO_VERSION
```

Comment:
I vaguely remember `ci-operator` complaining about a `build_arg` which was specified in its configuration but missing in the dockerfile, but I don't know whether it was fatal. But we will soon know if that's the case...

Reply:
Looks like it's not an issue.